src/storage/SimpleResourceStore.ts

```diff
@@ -1,24 +1,18 @@
 import arrayifyStream from 'arrayify-stream';
-import { BinaryRepresentation } from '../ldp/representation/BinaryRepresentation';
+import { DATA_TYPE_BINARY } from '../util/ContentTypes';
 import { ensureTrailingSlash } from '../util/Util';
 import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
-import { Quad } from 'rdf-js';
-import { QuadRepresentation } from '../ldp/representation/QuadRepresentation';
 import { Representation } from '../ldp/representation/Representation';
-import { RepresentationPreferences } from '../ldp/representation/RepresentationPreferences';
 import { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
 import { ResourceStore } from './ResourceStore';
 import streamifyArray from 'streamify-array';
-import { StreamWriter } from 'n3';
-import { UnsupportedMediaTypeHttpError } from '../util/errors/UnsupportedMediaTypeHttpError';
-import { CONTENT_TYPE_QUADS, DATA_TYPE_BINARY, DATA_TYPE_QUAD } from '../util/ContentTypes';
 
 /**
- * Resource store storing its data as Quads in an in-memory map.
+ * Resource store storing its data in an in-memory map.
  * All requests will throw an {@link NotFoundHttpError} if unknown identifiers get passed.
  */
 export class SimpleResourceStore implements ResourceStore {
```

Comment:
To be renamed into `InMemoryStore` at some point.

Reply:
Will require plenty of changes still before it counts as a "valid" store.